components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -815,7 +802,7 @@ void AdsServiceImpl::MigratePrefs() const {
 bool AdsServiceImpl::MigratePrefs(
     const int source_version,
     const int dest_version,
-    const bool is_dry_run) const {
+    const bool is_dry_run) {
```

Comment:
why const was removed?

Reply:
due to compiler errors after code changes

Reply:
Probably because `SetIntegerPref` isn't const (and doesn't look like it can be, unfortunately).